--- ghex-2.8.2/configure.in.orig	2006-03-07 21:05:47.000000000 +0100
+++ ghex-2.8.2/configure.in	2006-03-18 18:20:38.000000000 +0100
@@ -57,7 +57,7 @@
 GETTEXT_PACKAGE=ghex-2.0
 AC_SUBST(GETTEXT_PACKAGE)
 AC_DEFINE_UNQUOTED(GETTEXT_PACKAGE, "$GETTEXT_PACKAGE", [Gettext package name.])
-ALL_LINGUAS="am az be bg ca cs da de el en_CA en_GB es et eu fi fr ga gl hr hu id it ja ko ml ms nb ne nl nn no pa pl pt pt_BR ro ru rw sk sl sq sr sr@Latn sv tr uk vi wa zh_CN zh_TW"
+ALL_LINGUAS="am az be bg ca cs da de el en_CA en_GB es et eu fi fr ga gl hr hu id it ja ko ml ms nb ne nl nn pa pl pt pt_BR ro ru rw sk sl sq sr sr@Latn sv tr uk vi wa zh_CN zh_TW"
 AM_GLIB_GNU_GETTEXT
 
 AC_CHECK_LIB(popt, poptGetArgs, [POPT_LIBS="-lpopt"],
