diff -urN ghex-2.2.0.orig/Makefile.in ghex-2.2.0/Makefile.in
--- ghex-2.2.0.orig/Makefile.in	Tue Jan  7 13:03:09 2003
+++ ghex-2.2.0/Makefile.in	Tue Jan  7 13:45:39 2003
@@ -505,10 +505,6 @@
 #@MAINT@macros/macros.dep: macros/Makefile.am
 #@MAINT@	cd macros && $(MAKE) macros.dep
 
-#Install the ghex2 schema file
-install-data-local:
-	GCONF_CONFIG_SOURCE=$(GCONF_SCHEMA_CONFIG_SOURCE) $(GCONFTOOL) --makefile-install-rule $(srcdir)/$(schema_DATA)
-
 # Tell versions [3.59,3.63) of GNU make to not export all variables.
 # Otherwise a system limit (for SysV at least) may be exceeded.
 .NOEXPORT:
