diff -urN ghex-2.2.0.orig/help/C/ghex-C.omf ghex-2.2.0/help/C/ghex-C.omf
--- ghex-2.2.0.orig/help/C/ghex-C.omf	Tue Jan  7 13:03:09 2003
+++ ghex-2.2.0/help/C/ghex-C.omf	Tue Jan  7 13:03:41 2003
@@ -27,7 +27,7 @@
       user's guide
     </type>
     <format mime="text/xml" dtd="-//OASIS//DTD DocBook XML V4.1.2//EN"/>
-    <identifier url="ghex.xml"/>
+    <identifier url="ghex2.xml"/>
     <language code="C"/>
     <relation seriesid="2d280b70-2d11-11d6-8dca-89519bb1454b"/>
     <rights type="GNU FDL" license.version="1.1" holder="Sun Microsystems"/>
diff -urN ghex-2.2.0.orig/help/ja/ghex-ja.omf ghex-2.2.0/help/ja/ghex-ja.omf
--- ghex-2.2.0.orig/help/ja/ghex-ja.omf	Tue Jan  7 13:03:09 2003
+++ ghex-2.2.0/help/ja/ghex-ja.omf	Tue Jan  7 13:04:32 2003
@@ -13,7 +13,7 @@
     </date>
     <subject category="GNOME|Utilities"/>
     <format mime="text/sgml"/>
-    <identifier url="ghex.sgml"/>
+    <identifier url="ghex2.sgml"/>
     <language code="ja"/>
     <relation seriesid="6317f7e2-a88e-11d6-9c18-b2a33b44b6cd"/>
   </resource>
