--- ghex-2.8.2/Makefile.in.orig	2006-03-07 20:54:48.000000000 +0100
+++ ghex-2.8.2/Makefile.in	2006-03-18 18:15:59.000000000 +0100
@@ -799,9 +799,6 @@
 
 @INTLTOOL_DESKTOP_RULE@
 
-#Install the ghex2 schema file
-install-data-local:
-	GCONF_CONFIG_SOURCE=$(GCONF_SCHEMA_CONFIG_SOURCE) $(GCONFTOOL) --makefile-install-rule $(srcdir)/$(schema_DATA)
 # Tell versions [3.59,3.63) of GNU make to not export all variables.
 # Otherwise a system limit (for SysV at least) may be exceeded.
 .NOEXPORT:
