diff -urN aa/ghex-2.5.2/configure.in ghex-2.5.2/configure.in
--- aa/ghex-2.5.2/configure.in	2003-12-29 14:24:17.000000000 +0100
+++ ghex-2.5.2/configure.in	2004-04-12 14:50:08.892181368 +0200
@@ -56,7 +56,7 @@
 GETTEXT_PACKAGE=ghex-2.0
 AC_SUBST(GETTEXT_PACKAGE)
 AC_DEFINE_UNQUOTED(GETTEXT_PACKAGE, "$GETTEXT_PACKAGE", [Gettext package name.])
-ALL_LINGUAS="am az be bg ca cs da de el en_GB es et eu fi fr ga gl hr hu it ja ko ml ms nl nn no pl pt pt_BR ro ru sk sl sr sr@Latn sv tr uk vi wa zh_CN zh_TW"
+ALL_LINGUAS="am az be bg ca cs da de el en_GB es et eu fi fr ga gl hr hu it ja ko ml ms nb nl nn pl pt pt_BR ro ru sk sl sr sr@Latn sv tr uk vi wa zh_CN zh_TW"
 AM_GLIB_GNU_GETTEXT
 
 AC_CHECK_LIB(popt, poptGetArgs, [POPT_LIBS="-lpopt"],
